frontend/package.json

```diff
@@ -33,7 +33,10 @@
     "test:fix": "eslint --ext .json,.js,.ts,.jsx,.tsx ./src --fix",
     "test:lint": "eslint --max-warnings 0 --ext .json,.js,.ts,.jsx,.tsx ./src",
     "test:local": "jest --watchAll",
-    "test:type-check": "tsc --noEmit"
+    "test:type-check": "tsc --noEmit",
+    "test:storybook": "test-storybook --verbose",
+    "storybook": "start-storybook -p 6006 -s ./config",
```

Comment:
Can we get this added to the root README? I'd imagine it can go right after `npm test`

Reply:
i didnt right away just so we could slowly adopt this

Reply:
Might as well add it with this PR -- shouldn't be too much documentation needed to just note it exists.